app/helpers/application_helper.rb

```diff
@@ -334,4 +328,25 @@ def get_thematical_areas
       "cards": thematical_page.children.published
     }
   end
+
+  def get_footer_links
+    def make_links slug_array
```

Comment:
Also please do not define nested methods.
If you know a method is only going to be used from other methods, define this method as private.

In ruby you can start a list of private methods by just using `private` before those definitions, like:

```
  private

  def my_method
  end

  def another_method
   end
```